README.md

```diff
@@ -1,2 +1,56 @@
 # Hello-world
 This is my first Microverse study repo.
+
+
+## Built With
+
+- Major languages - Html and CSS
+
+## Live Demo
+
+[Live Demo Link](https://github.com/white3d/Hello-world)
+
+
+## Getting Started
+
+
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
Kindly make sure there are no empty sections. This will make sure that your README appear more professional.
Please complete them or remove the empty sections completely

Reply:
Sure... I will adjust that ASAP.